python-threatexchange/threatexchange/fetcher/apis/tests/test_stopncii.py

```diff
@@ -25,27 +21,27 @@ def fetcher(api: StopNCIIAPI):
     return signal_exchange
 
 
-def test_fetch(fetcher: SignalExchangeAPI, monkeypatch: pytest.MonkeyPatch):
+def test_fetch(fetcher: StopNCIISignalExchangeAPI, monkeypatch: pytest.MonkeyPatch):
```

Comment:
non-blocking. We've used [freezegun](https://github.com/spulec/freezegun) for moving through time in HMA. If the testing gets any more complex, freezegun's API is much more readable and does not rely on magic numbers.

Reply:
I might attempt to convince you in the other direction - I've found the py.test fixture functionality pretty usable, and just has a short barrier to understanding it.